deeprank/learn/DataSet.py

```diff
@@ -285,18 +285,19 @@ def process_dataset(self):
         self.get_pairing_feature()
 
         # get grid shape
-        self.get_grid_shape()
+        if self.grid_shape is None:
```

Comment:
Does it make sense to make `grid_shape` private? i.e. remove the argument `grid_shape`. Looking at the code after a long time, the argument `grid_info` and `grid_shape` are confusing to me.